lib/canvas.js

```diff
@@ -35,6 +36,9 @@ Canvas.prototype = {
     assert(Canvas.supportedFormats.indexOf(format) !== -1,
       '`gmsmith` doesn\'t support exporting a "' + format + '". Please export either a `png` or `jpeg`');
 
+    // Extent to given size
+    canvas.gravity('NorthWest').extent(this.width, this.height, '!');
```

Comment:
I would believe it; I recall having a lot of issues when I initially wrote the `gm` integration =/ If it doesn't work out, can we add a comment like:

```js
// DEV: We set width/height via `extent` as Graphics Magick/ImageMagick don't respect transparency otherwise
var canvas = gmsmith.gm(1, 1, 'transparent'); 
```